automations/js/src/count_user_reviewable_prs.js

```diff
@@ -78,13 +77,26 @@ query ($repoOwner: String!, $repo: String!, $cursor: String) {
       }
     }
 
+    let should_alert = false
```

Comment:
We use camel case for JavaScript variable names.

I wonder why prettier/eslint doesn't catch this? :thinking: 

```suggestion
    let shouldAlert = false
```

Reply:
Oops 😅 force of habit!